prow/tide/github.go

```diff
@@ -246,10 +246,9 @@ func (gi *GitHubProvider) mergePRs(sp subpool, prs []CodeReviewCommon, dontUpdat
 
 	for i, pr := range prs {
 		log := log.WithFields(pr.logFields())
-		mergeMethod, err := gi.prMergeMethod(&pr)
-		if err != nil {
-			log.WithError(err).Error("Failed to determine merge method.")
-			errs = append(errs, err)
+		mergeMethod := gi.prMergeMethod(&pr)
+		if mergeMethod == nil {
+			log.Error("Multiple merge method labels are not supported.")
 			failed = append(failed, pr.Number)
```

Comment:
We still need to add an error to `errs` in addition to adding the PR to `failed` so the error handling starting around line 288 will work properly.

Reply:
yeah, I had doubts about that. First I added, then removed. I'll correct it tomorrow though.